ngnix-tut.md

```diff
@@ -0,0 +1,8 @@
+1. Make sure you have a docker-compose.common.yml file with the common service configurations.
```

Comment:
not sure what is this file for

Reply:
Hey @talboren 
To avoid duplication services like keep-frontend and keep-backend are extended from `docker-compose.common.yml.` They inherit all the configurations defined in `docker-compose.common.yml.`